app/controllers/tweets_controller.rb

```diff
@@ -21,25 +20,24 @@ class TweetsController < ApplicationController
     if params[:content] == ""
       redirect to "/tweets/new"
     else
-      user = User.find_by_id(session[:user_id])
-      @tweet = Tweet.create(:content => params[:content], :user_id => user.id)
+      @tweet = Tweet.create(:content => params[:content], :user_id => current_user.id)
```

Comment:
Thanks Gabe,

But we should actually change this to be `current_user.tweets.create(content: params[:content])`